Controller/Api/MessageController.php

```diff
@@ -64,7 +64,7 @@ public function __construct(MessageManagerInterface $messageManager, FormFactory
      *
      * @QueryParam(name="page", requirements="\d+", default="1", description="Page for message list pagination")
      * @QueryParam(name="count", requirements="\d+", default="10", description="Number of messages by page")
-     * @QueryParam(name="orderBy", array=true, requirements="ASC|DESC", nullable=true, strict=true, description="Order by array (key is field, value is direction)")
+     * @QueryParam(name="orderBy", map=true, requirements="ASC|DESC", nullable=true, strict=true, description="Order by array (key is field, value is direction)")
```

Comment:
Actually, you can do this in a BC-manner. See [this PR](https://github.com/sonata-project/SonataMediaBundle/pull/1155) from @lemoraggia this morning.